File: clients/java/examples/admin-create-database.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.AstraDBAdmin;
import com.dtsx.astra.sdk.db.domain.CloudProviderType;

public class Example {
  public static void main(String[] args) {
    DataAPIClient client = new DataAPIClient("**APPLICATION_TOKEN**");

    AstraDBAdmin admin = client.getAdmin();

    admin.createDatabase("**DATABASE_NAME**", CloudProviderType.GCP, "us-east1");
  }
}

File: clients/typescript/examples/admin-create-database.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

const client = new DataAPIClient("**APPLICATION_TOKEN**");

const admin = client.admin();

(async function () {
  await admin.createDatabase({
    name: "**DATABASE_NAME**",
    cloudProvider: "GCP",
    region: "us-east1",
  });
})();
